src/shell/layout/tiling/mod.rs

```diff
@@ -3843,6 +3841,10 @@ impl TilingLayout {
 
         Ok((window_elements, popup_elements))
     }
+
+    pub(crate) fn update_gaps(&mut self, gaps: (i32, i32)) {
+        self.gaps = gaps;
+    }
```

Comment:
This is somewhat inconsistent. The `IcedElement` reloads the theme, but although `theme::gaps()` exists, this gets set instead of a similar `update_gaps`-function. (Or more generally `update_theme`.)

Not sure what solution would be cleanest, but I would like it to be consistent.